src/components/BlogPost/BlogPost.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { Grid, Paper, Container } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+const styles = (theme) => ({
+  content: {
+    flexGrow: 1,
+  },
+  description: {
+    padding: theme.spacing(2),
+    textAlign: 'left',
+    color: theme.palette.text.secondary,
+  },
+  image: {
```

Comment:
Once you learn about media queries from my other comment about grid order, you should play around with manipulating the style with different device sides. This is a little bit more work but its good practice, makes you a better frontend developer when you consider how your site/app looks on different devices.